docker/kallisto/Dockerfile

```diff
@@ -11,7 +11,13 @@ RUN apt update && apt install -y \
   cmake \
   curl \
   libhdf5-dev \
-  samtools
+  libncurses5-dev \
```

Comment:
Is there a way to indicate which tool these are dependencies of? Otherwise people will probably keep them in here forever, even if, for example, we remove samtools later, because people won't be sure what they're for.

Reply:
Without looking at other mature big dockers, I come up with two ideas to address this:
1. Elaborate comments in dockerfiles.
2. Use separate `apt install` command for each tool and its dependencies. (like one command block  for star and another for samtools)
But I will check if there are any other better solutions used in other big dockers.